theme/basics/template-hierarchy.md

```diff
@@ -1,419 +1,184 @@
-2. `home.php` - WordPress が `front-page.php` を見つけられず、且つ「**フロントページの表示**」セクションで「**最新の投稿**」が設定されている場合、`home.php` を参照します。加えて、WordPress は「**フロントページの表示**」セクションで「**投稿ページ**」が設定されている場合にもこのファイルを参照します。
-3. `page.php` – 「**フロントページの表示**」セクションで**フロントページ**が設定されている場合。
-4. `index.php` – 「**フロントページの表示**」セクションで「**最新の投稿**」が設定されており、且つ `home.php` が存在しない場合。または**フロントページ**が設定されているが `page.php` が存在しない場合。
 
-<!--
 As you can see, there are a lot of rules to what path WordPress takes. Using the chart above is the best way to determine what WordPress will display.
--->
-このように、WordPressがどのような経路をとるかには多くのルールがあります。上の図を使用することは、WordPress が何を表示するかを特定する最良の方法です。
 
-<!--
 ### Privacy Policy Page display
--->
-### プライバシーポリシーページ表示
 
-<!--
 The `privacy-policy.php` template file is used to render your site’s Privacy Policy page. The Privacy Policy page template takes precedence over the static page (`page.php`) template. If the `privacy-policy.php` file does not exist, WordPress will either use the `page.php` or `singular.php` files depending on the available templates. If neither of those files exist, it will use the `index.php` file.
--->
-`privacy-policy.php` テンプレートファイルは、あなたのサイトのプライバシーポリシーページをレンダリングするために使われます。プライバシーポリシーページのテンプレートは、静的ページ（`page.php`）のテンプレートよりも優先されます。`privacy-policy.php` ファイルが存在しない場合、WordPress は利用可能なテンプレートに応じて `page.php` または `singular.php` ファイルを使用します。これらのファイルが存在しない場合は、`index.php`ファイルを使用します。
 
-<!--
 1.  `privacy-policy.php` – Used for the Privacy Policy page set in the **Change your Privacy Policy page** section of Settings → Privacy.
 2.  `custom template file` – The [page template](https://developer.wordpress.org/themes/template-files-section/page-template-files/) assigned to the page. See `get_page_templates()`.
 3.  `page-{slug}.php` – If the page slug is `privacy`, WordPress will look to use `page-privacy.php`.
 4.  `page-{id}.php` – If the page ID is 6, WordPress will look to use `page-6.php`.
 5.  `page.php`
 6.  `singular.php`
 7.  `index.php`
--->
-1.  `privacy-policy.php` – 「設定」→「プライバシー」の「**プライバシーポリシーページの変更**」で設定したプライバシーポリシーページに使用されます。
-2.  `カスタムテンプレートファイル` – ページに割り当てられた[ページテンプレート](https://developer.wordpress.org/themes/template-files-section/page-template-files/)です。詳しくは `get_page_templates()` をご覧ください。
-3.  `page-{slug}.php` – ページスラッグが `privacy` の場合、WordPress は `page-privacy.php` の使用を検討します。
-4.  `page-{id}.php` – ページ ID が 6 の場合、WordPressは `page-6.php` を使おうとします。
-5.  `page.php`
-6.  `singular.php`
-7.  `index.php`
 
-<!--
 ### Single Post
--->
-### 個別投稿
 
-<!--
 The single post template file is used to render a single post. WordPress uses the following path:
--->
-個別投稿のテンプレートファイルは、1 つの投稿を表示するために使用されます。WordPress では、以下の順で表示されます:
 
-<!--
 1.  `single-{post-type}-{slug}.php` – (Since 4.4) First, WordPress looks for a template for the specific post. For example, if [post type](https://developer.wordpress.org/themes/basics/post-types/) is `product` and the post slug is `dmc-12`, WordPress would look for `single-product-dmc-12.php`.
```

Comment:
<!--
1.  `single-{post-type}-{slug}.php` – (Since 4.4) First, WordPress looks for a template for the specific post. For example, if [post type](https://developer.wordpress.org/themes/basics/post-types/) is `product` and the post slug is `dmc-12`, WordPress would look for `single-product-dmc-12.php`.
2.  `single-{post-type}.php` – If the post type is `product`, WordPress would look for `single-product.php`.
3.  `single.php` – WordPress then falls back to `single.php`.
4.  `singular.php` – Then it falls back to `singular.php`.
5.  `index.php` – Finally, as mentioned above, WordPress ultimately falls back to `index.php`.
-->
1.  `single-{post-type}-{slug}.php` – まず、WordPress は特定の記事のテンプレートを探します(4.4 以降) 。例えば、[投稿タイプ](https://developer.wordpress.org/themes/basics/post-types/)が `product` で、記事のスラッグが `dmc-12` の場合、WordPress は `single-product-dmc-12.php` を探します。
2.  `single-{post-type}.php` – 投稿タイプが `product` であれば、WordPress は `single-product.php` を探します。
3.  `single.php` – 次に WordPress は `single.php` へフォールバックします。
4.  `singular.php` – その次に `singular.php` へフォールバックします。
5.  `index.php` – 最終的には、前述のとおり、WordPress は `index.php` へフォールバックします。